.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,67 @@
+---
+name: 🐛 Issue report
+about: I hit an error when I tried to use this plugin.
+---
+
+### Describe the issue
+<!--
+  A clear and concise description of what the issue is.
+
+  For example - When I try to authorize the reader with the correct location and oauth param it returns an exception with message "authorization failed unexpectly.". Here is the full error information:
+  ```
+  // paste your error log here
+  ```
+-->
+
+### To Reproduce
+<!--
+  Steps to reproduce the issue.
+
+  For example - 
+  1. Initialize the SDK
```

Comment:
This will just be a template and they can see the example from comment, but now shown once published. 

A template is used like this: https://github.com/flutter/flutter/issues/new/choose

Reply:
Okay. Just wanted to make sure the example list would be seen in an environment that would render the list properly :)